src/dsp_tools/analyse_xml_data/models_xml_to_graph.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+
+
+@dataclass
+class ResptrLink:
+    """This class represents a link between two resources."""
+
+    subject_id: str
+    object_id: str
+
+
+@dataclass
+class XMLLink:
+    """This class represents a link between a resource and an XMl text which contains links to other resources."""
```

Comment:
I don't fully understand this. The variable names `subject_id` and `object_link_ids` suggest that this class represents 1 resource that contains 1 xml-text with several links to other resources.

If my assumption is correct, I would rephrase the docstring: `This class represents a resource with an XML text which contains links to other resources.`